### ISSUE TITLE
1.3 Installer verification

### ISSUE THREAD

**Issue by adamcfraser** [couchbase/sync_gateway#1911]
Following target OS's and SG versions have been tested.
### Sync Gateway Installation on a clean OS image.
- [x] Debian 8
- [x] Ubuntu 12
- [x] Ubuntu 14
- [x] RedHat/CentOS 5
- [x] RedHat/CentOS 6
- [x] RedHat/CentOS 7
- [x] Darwin/OSX/macOS (El Capitan)
- [x] Windows (Windows 8, 10, 2012 server tested)
### Sync Gateway Upgrades from SG 1.2.1.
- [ ] Debian 8 (OS not supported before 1.3)
- [x] Ubuntu 12 (dpkg -i)
- [x] Ubuntu 14 (dpkg -i)
- [x] RedHat/CentOS 5 (rpm -U)
- [x] RedHat/CentOS 6 (rpm -U)
- [x] RedHat/CentOS 7 (rpm -U)
- [x] Darwin/OSX/macOS (El Capitan)
- [x] Windows 8/10 (1.2.1 enterprise install failed, upgraded from 1.2.0 install), (Windows 2012 server not tested)
### SG Accel Installation on a clean OS image.
- [x] Debian 8
- [x] Ubuntu 12
- [x] Ubuntu 14
- [x] RedHat/CentOS 5
- [x] RedHat/CentOS 6
- [x] RedHat/CentOS 7
- [x] Darwin/OSX/macOS (El Capitan)
- [x] Windows (Windows 8/10/2012 server)
### sgcollect_info installed with Sync gateway
- [x] Debian 8
- [x] Ubuntu 12
- [x] Ubuntu 14
- [x] RedHat/CentOS 5
- [x] RedHat/CentOS 6
- [x] RedHat/CentOS 7
- [x] Darwin/OSX/macOS (El Capitan)
- [x] Windows (Windows 8/10/2012 server)
### sgcollect_info installed with SG Accel
- [x] Debian 8
- [x] Ubuntu 12
- [x] Ubuntu 14
- [x] RedHat/CentOS 5
- [x] RedHat/CentOS 6
- [x] RedHat/CentOS 7
- [x] Darwin/OSX/macOS (El Capitan)
- [ ] Windows (Windows 8/10/2012 server) (no tools folder installed)
### sgcollect_info validation

Run sgcollect_info, validate .zip contents (check for sensitive data), validate .zip upload to support 

Tested with  build 266
- [x] Debian 8
- [x] Ubuntu 12
- [x] Ubuntu 14
- [x] RedHat/CentOS 5
- [x] RedHat/CentOS 6
- [x] RedHat/CentOS 7
- [x] Darwin/OSX/macOS (El Capitan)
- [x] Windows (Windows 8/10/2012 server)

**Comment by ajres** [comment]
Do we want to support Debian? We have an open ticket #1517

**Comment by ajres** [comment]
There is a regression with SG 1.3.0 on RedHat/CentSO 5, the rpm package fails to install with the following message:

```
rpm -i couchbase-sync-gateway-enterprise_1.3.0-234_x86_64.rpm
error: Failed dependencies:
    rpmlib(FileDigests) <= 4.6.0-1 is needed by couchbase-sync-gateway-1.3.0-234.x86_64
    rpmlib(PayloadIsXz) <= 5.2-1 is needed by couchbase-sync-gateway-1.3.0-234.x86_64
```

SG 1.2.1 .rpm installed without error

`rpm -i couchbase-sync-gateway-enterprise_1.2.1-4_x86_64.rpm`

**Comment by ajres** [comment]
Issue with Windows community installer (can't see enterprise installer build yet), license.txt file is missing, I'll files a separate issue.

**Comment by michaelkwok** [comment]
The enterprise failed unit test.  Just curious, did you get an email notification on the failure?

**Comment by michaelkwok** [comment]
Does the rpm installation only failed with Centos 5?  I assume it works fine for newer Centos or it would've failed QE functional test which is triggered on every success build.  The current 1.3.0 Centos builds come from a  Centos 7 build server.

**Comment by ajres** [comment]
Yes the rpm only fails on CentOS 5, this is a regression, was 1.2.1 built on on older CentOS version?

**Comment by michaelkwok** [comment]
1.2.1 was built on Centos 6.5.  Zack had requested to switch to Centos 7 for 1.3.0 release.  So it's good you caught this.

**Comment by michaelkwok** [comment]
Regarding deb issue in ticket #1517 reported the issue on build 1.2.0-65. We released build 1.2.0-83 so you can give it a try.  Maybe build 1.2.0-65 had an issue.

**Comment by michaelkwok** [comment]
The license issue is related to PR #1916.  It should be fixed once the pr is merged to master.

**Comment by ajres** [comment]
Debian 8

[sgcollect_info stdout](https://gist.github.com/ajres/62c3f64a6baabb016233b1ff5108157e)

**Comment by ajres** [comment]
With build #266 uploading the sgcollect_info  .ZIP file to s3 fails on Windows.

It looks like this is because the do_upload_and_exit task is still trying to run the external curl command.

**Comment by ajres** [comment]
@tleyden has opened #1990, closing this ticket out now.
